ecosystem/indexer-grpc/indexer-test-transactions/build.rs

```diff
@@ -1,36 +1,53 @@
 // Copyright (c) Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
-
-// build.rs
 use std::{env, fs, path::Path};
 
 fn main() {
     let out_dir = env::var("OUT_DIR").unwrap();
     let dest_path = Path::new(&out_dir).join("generate_transactions.rs");
 
     let mut all_transactions_code = String::new();
+    let mut name_function_code = String::new();
```

Comment:
thoughts: i feel the builder pattern may work here better; the string construction pattern may make debug very hard